docs/browse-list.md

```diff
@@ -20,7 +20,23 @@ Expected time: 5.5 - 6 hours.
 
 ## Troubleshooting
 
-When browse lists fail to generate and would like to run the rake task before it is scheduled:
+When **subject lists**, which is used in `Subject (browse)` in Orangelight, fail to generate and would like to run the rake task before it is scheduled: SSH as `deploy` user to the machine used to produce the browse lists (`catalog-indexer1|2|3`):
+
+- Run the first rake task to generate the CSV file from solr data.
+- `cd /opt/orangelight/current`
+- `OL_DB_PORT=5432 bundle exec rake browse:subjects`
+
+Expected time 1.5-2 hours.
+
+- Run the second rake task to upload the CSV file in the postgres table.
+
+- `OL_DB_PORT=5432 bundle exec rake browse:load_subjects`
+
+Expected time: Less than 30 minutes.
+
+See note about `schedule.rb` in the Troubleshooting section below to find out the exact machine.
```

Comment:
This line is confusing because we're in the troubleshooting section.

Reply:
Maybe move the "how to know which machine to use" section up above the list-specific sections and make it more general?